.github/workflows/loadtest.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       - name: Install xk6
         run: |
           go install go.k6.io/xk6/cmd/xk6@latest
-          xk6 build --with github.com/distribworks/xk6-ethereum@5c6e782669953f1e5d1f44509e610fb2e3d22238
+          xk6 build --with github.com/distribworks/xk6-ethereum@26fa194b68544e53aecfe5b29aa785c4de7a52f1
```

Comment:
WDYT about using the latest release? @vcastellm created it yesterday (https://github.com/distribworks/xk6-ethereum/releases/tag/v1.0.2).